src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CurrentUserOnly.netcoreapp.cs

```diff
@@ -116,5 +116,37 @@ public static void CreateMultipleServers_ConnectMultipleClients_MultipleThreads(
 
             Task.WaitAll(tasks.ToArray());
         }
+
+        [Fact]
+        public static void CreateMultipleConcurrentServers_ConnectMultipleClients()
```

Comment:
Does this test work on unix?

Reply:
Nope...

Reply:
It is failing on a Debug.Assert inside SharedServer.Get: it wants max number of services to be positive, however, we are keeping the sentinel value -1 (MaxAllowedServerInstances). So it is working as intended but we either change the assert or the max number of services to int.MaxValue if the sentinel value was found. I will update the PR.